Resources/public/templates/testoneview.hbt

```diff
@@ -0,0 +1,15 @@
+<div>
+    <header class="ez-page-header">
```

Comment:
To samo co wyżej: https://github.com/gbalcewicz/PlatformUIBundle/pull/1/files#r41963948